### PR TITLE
Adding cordova mobile app. Updated .gitignore.

### PR DESCRIPTION
**Pull request by gardner**
*EDIT:* This pull request correctly requests a merge into the ``beta`` branch.

This feature was branched from ``beta``.  It wraps cobudget into a cordova mobile app. To run the app on the iOS simulator please ensure that Xcode is installed and that a local api server is running at localhost:3000 then:

    npm install -g cordova
    cd cobudget-ui/mobile/
    cordova platform add ios
    cd .. ; gulp build ; cd mobile ; cordova run ios
 
After any code changes the angular app must be rebuilt before running ``cordova run ios``

## Happy Halloween!

simulator screen shot oct 31 2015 7 59 51 pm

### CODE REVIEW

**Comment by ahdinosaur** [comment]
:tada:

**Comment by mixmix** [comment]
wow, this is exciting, thanks @gardner . 
We're pushing to finish a bunch of work in our [current sprint](https://trello.com/b/Ca4BpYOz/cobudget-sprint-board) . We're running super lean, and with a bit of time pressure.

We'll be planning our next sprint on thursday, I've added [a card in the backlog](https://trello.com/c/57MuDBrK/326-mobile-wrapping-pull-request) so we discuss where this piece you've offered fits into the plan.

**Comment by mixmix** [comment]
Hi @gardner sorry for the slow reply :turtle: 
I'd love to see this in action. Right now we've got a huge number of things we're working to get out the door to get a really minimal easy app for more people. (our sprint board : https://trello.com/b/Ca4BpYOz/cobudget-sprint-board) 

In short, we're a way away from being able to do the work to fold this in to `beta` directly and support it.

Do you have any thoughts about how we can integrate this without having to take on and maintain 257 file changes ? Ideas off the top of my head : 

- perhaps a separate repo, symlinked into place, or a git submodule would serve here? 
- a seperate `cordova` branch that we just maintain

**Comment by mixmix** [comment]
Is it simple to compile a package to try out directly on an Android / iPhone ? And can you produce those for us to try out?
(again, I'm reallllyyy curious and want to spin this up but I can't afford time to explore this attractive rabbit hole)

Look forward to seeing this in action. 
Thanks again for your work, it's magic   (ﾉ´ヮ´)ﾉ*:･ﾟ✧

**Comment by gardner** [comment]
Hi @mixmix,
  I understand your conflicted sense of urgency. I have been talking with @derekrazo about this pull request and he mentioned that there are a few UI things to fix before we push this into a beta channel to endure a more public scrutiny. Please feel comfortable in knowing that I will work with him to resolve these UI issues and then push an easily downloadable package for Android and iOS. I hope that having this available on all the developer devices will reduce friction and increase collaboration. After we've gotten to that point it will make more sense for the developers to start analyzing the state of the mobile app and prioritizing the backlog accordingly to ultimately create a great cobudget mobile app.

In short: please feel free to ignore this pull request for a couple of weeks. Things are a-brewing.

**Comment by euglazer** [comment]
thx @gardner !

you are srsly the best :+1:

**Comment by mixmix** [comment]
thanks for your clear communication and understanding :)

**Comment by derekrazo** [comment]
Some flossing and fixy stuff to figure out before deploying beta mobile

__MVP__

[ ] app shouldn't hide the carrier, battery and time at the top of the phone (not sure if this is possible)
[ ] mobile app should have a logo
whatiscobudget
[ ] fix [issue](https://trello.com/c/T08qMtxA/281-stuck-on-funders-page-when-opening-more-button-on-a-person) with getting stuck in menus on funders page

__Next phase__

[ ] mobile app should push notifications for new activity in groups (to be better defined)
